ingress-nginx.tf

```diff
@@ -19,32 +19,37 @@ locals {
   ingress_nginx_config = templatefile(
     "${path.module}/files/ingress-nginx/values.yaml",
     {
-      vpc_cidr_block = var.vpc_cidr_block
+      vpc_cidr_block         = var.vpc_cidr_block
       aws_load_balancer_name = local.ingress_nginx_nlb_name
     }
   )
 }
 
-resource "kubernetes_namespace" "ingress_nginx" {
+resource "kubernetes_namespace_v1" "ingress_nginx" {
```

Comment:
Remove this resource and use the namespace create in the module. We probably need to bump the version of this module to `3.0.0`.